docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -1179,6 +1183,29 @@ The table above shows the following information about the endpoint:
 * SERVICE DESCRIPTION : Url of the service description owner of the endpoint.
 * TYPE : Type of the service description owner of the endpoint.
 
+##### 4.2.8.4 Endpoint update
+
+* Access rights: XROAD_SERVICE_ADMINISTRATOR
```

Comment:
Other "endpoint" operations require also XROAD_SYSTEM_ADMINISTRATOR, this one does not?

Reply:
No, the others require the XROAD_SYSTEM_ADMINISTRATOR because they use the configuration file, and when using the configuration file we perform a get status request that require this role, but for the commands with params is not required